apps/web/app/api/(internal)/pipeline/route.ts

```diff
@@ -226,6 +227,8 @@ export const POST = async (request: Request) => {
       }
     });
   }
+  // Send telemetry events
+  await sendTelemetryEvents();
```

Comment:
I would suggest we only send telemetry on `responseCreated` event in the pipeline to avoid unnecessary checks for following response update events / partial responses.